target/linux/ipq806x/image/Makefile

```diff
@@ -82,6 +82,21 @@ define Device/ZyXELImage
 	IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-to $$$$(BLOCKSIZE) | sysupgrade-tar rootfs=$$$$@ | append-metadata
 endef
 
+define Device/askey_rt4230w-rev6
+        $(call Device/LegacyImage)
+        DEVICE_VENDOR := Askey
+        DEVICE_MODEL := RT4230W
+        DEVICE_VARIANT := REV6
+        SOC := qcom-ipq8065
+        BLOCKSIZE := 128k
+        PAGESIZE := 2048
+        DEVICE_PACKAGES := ath10k-firmware-qca9984-ct
+        KERNEL_IN_UBI := 1
+        IMAGES := nand-sysupgrade.bin
+        IMAGE/nand-sysupgrade.bin := sysupgrade-tar | append-metadata
```

Comment:
Please use tabs for indent here like the other devices do.

Reply:
Done